enterprise/frontend/src/metabase-enterprise/sandboxes/components/EditSandboxingModal/EditSandboxingModal.tsx

```diff
@@ -133,16 +133,17 @@ const EditSandboxingModal = ({
 
   return (
     <div>
-      <h2 className={CS.p3}>{t`Grant sandboxed access to this table`}</h2>
+      <h2 className={CS.p3}>{t`Restrict access to this table`}</h2>
 
       <div>
         <div className={cx(CS.px3, CS.pb3)}>
-          <div className={CS.pb3}>
-            {t`When users in this group view this table they'll see a version of it that's filtered by their user attributes, or a custom view of it based on a saved question.`}
+          <div className={CS.pb2}>
+            {t`When the following restriction rules are applied, users in this group will view a filtered or customized version of the table.`}
```

Comment:
If copy isn't set in stone, maybe:

`When the following rules are applied, users in this group will see a filtered or customized version of the table`